corehq/apps/app_manager/suite_xml/post_process/instances.py

```diff
@@ -258,6 +253,16 @@ def require_instances(entry, instances=(), instance_ids=()):
         if sorted_instances != entry.instances:
             entry.instances = sorted_instances
 
+    @staticmethod
+    def _should_ignore_instance(instance):
```

Comment:
this shouldn't be necessary anymore but fine to punt to a separate PR

Reply:
Updated in c81207d61fa though I don't totally understand. Is this true? => I can remove `_should_ignore_instance` because now instance URIs are unique, and previously we would ignore instances that could appear multiple times, for the sake of. not adding multiple instance declarations to the entry?

Reply:
Before we had eager initialisation of instances but these are only available after queries have been run.

Reply:
I still think this can be removed.